static/js/tools/choropleth/choropleth.ts

```diff
@@ -135,34 +134,27 @@ function handleMapClick(geo) {
 /**
  * Builds a redirect link given the fields to include from search params.
  * @param fields_to_include
- * @param from_api whether the url should be to the api or locally.
  */
-function build_choropleth_url(fields_to_include, from_api) {
-  //TODO(iancostello): Make this path relative.
-  let base_url = document.location.origin + document.location.pathname;
-  if (from_api) {
-    base_url += "/api";
-  }
-  base_url += "?";
+function build_choropleth_params(fields_to_include) {
+  let params = "?";
   for (const index in fields_to_include) {
     const arg_name = fields_to_include[index];
     const arg_value = url.searchParams.get(arg_name);
     if (arg_value != null) {
-      base_url += "&" + arg_name + "=" + arg_value;
+      params += "&" + arg_name + "=" + arg_value;
     }
   }
-  return base_url;
+  return params;
 }
 
 /**
  * Redirects the webclient to a particular geo. Handles breadcrumbs in redirect.
  * @param {string} geoId to redirect to.
  */
 function redirectToGeo(geoId) {
-  let base_url = build_choropleth_url(
-    ["statVar", "perCapita", "level", "mdom"],
-    false
-  );
+  let base_url = document.location.pathname;
```

Comment:
Do we need to worry about the case of "geoId"'s being the empty string?

Reply:
This is currently handled on the server side. In `server/routes/api/choropleth.py`, we have 
```
    if requested_geoId is None:
        return jsonify({"error": "Must provide a 'geoId' field!"}, 400)
```